packages/components/src/components/loaders/Spinner/Spinner.stories.tsx

```diff
@@ -1,22 +1,45 @@
 import { Meta, StoryObj } from '@storybook/react';
 import { Spinner as SpinnerComponent, SpinnerProps } from './Spinner';
-import { framePropsStory } from '../../common/frameProps';
 
 const meta: Meta = {
     title: 'Loaders/Spinner',
     component: SpinnerComponent,
 } as Meta;
 export default meta;
 
-export const Spinner: StoryObj<SpinnerProps> = {
+export const Refresh: StoryObj<SpinnerProps> = {
     args: {
-        ...framePropsStory.args,
         size: 50,
     },
     argTypes: {
         className: {
             control: false,
         },
-        ...framePropsStory.argTypes,
+    },
+};
+
+export const Success: StoryObj<SpinnerProps> = {
+    args: {
+        size: 50,
+        hasFinished: true,
+        isWithSpeadup: true,
```

Comment:
Could you rename this prop to something more clear? 🙏  
Maybe something like `hasInitialAnimation` 🤷‍♂️